app/schemas/ogc-api-coverages/1.0.0/ogc-api-coverages-v1.0.0.yml

```diff
@@ -666,6 +668,22 @@ components:
         type: array
         items:
           type: number
+    point:
+      name: point
+      in: query
+      description: |
+        Only collections that have a geometry that contains a spatial point are selected. The spatial point is provided as two numbers:
+        * Longitude, coordinate axis 1 * Latitude, coordinate axis 2
+        The coordinate reference system of the values is WGS84 longitude/latitude (http://www.opengis.net/def/crs/OGC/1.3/CRS84).
```

Comment:
I sort of remember while we were pairing you were reversing the points returned by CMR metadata. I could see that being confusing for end users - should we use latitude/longitude like the CMR?

Reply:
I forgot OGC might have a way for specifying the lat/lon order of points. You can disregard my comment about CMR and make sure we are matching the OGC specification.